tests/test_multiprocess.py

```diff
@@ -1,30 +1,43 @@
 from __future__ import unicode_literals
+
 import os
 import shutil
 import tempfile
-import time
 import unittest
 
-import prometheus_client
-from prometheus_client.core import *
-from prometheus_client.multiprocess import *
+from prometheus_client.core import (
```

Comment:
I'm not a big fan of these parens

Reply:
I agree the import blocks aren't very nice, I'd generally just bring in the module and prefix everything, but for tests specifically it's at least clear what it being used by stating each name.

Specifically for parens, support was added in 2.4 and this multiline style is generally preferred:
https://www.python.org/dev/peps/pep-0328/
https://stackoverflow.com/questions/14376900/is-there-a-recommended-format-for-multi-line-imports